multicluster/build/Dockerfile.deploy

```diff
@@ -0,0 +1,4 @@
+FROM gcr.io/distroless/static:nonroot as product
```

Comment:
this should be `Dockerfile.test` IMO, the name is confusing. I also think you should make it a separate target in the Makefile, so that `make docker-build` builds the "correct" image (the one with no prerequisites on local dependencies).

Reply:
I renamed build/Dockerfile.deploy to build/Dockerfile, the intention is that the same Dockerfile will be used for both build binaries, create product docker image.

hope this works for you.